### DISCUSSION TITLE
是否允许增加"自动排除所有用户级应用程序"开关

### DISCUSSION THREAD

**Discussion by qdsp6sw** [bmax121/APatch#441] [Q&A]
默认排除所有应用（尤其是系统应用）会对系统的性能带来较大的开销，但如果只对用户应用进行自动排除，开销会小很多，如果在大量安装APK/卸载-重新安装APK时UID有变化的情况下，会导致部分APP不知不觉的“被检测”，然后被拉黑，因此想讨论一下，是否有必要增加一个功能，可以自动排除所有用户级应用程序，并可以由用户设置是否要开启此功能？

**Comment by pomelohan** [comment]
个人觉得即便是用户应用，也没有使用全局排除的必要，因为用户应用的数量也不是少数，fork 的时候都会带来开销。UID 变化的问题我们会尽快解决。
